src/core/components/choice-card/styles.ts

```diff
@@ -135,10 +162,10 @@ export const choiceCard = ({
 
 	${from.mobileLandscape} {
 		margin: 0 ${space[2]}px 0 0;
+	}
 
-		&:last-child {
-			margin: 0;
-		}
+	&:last-child {
+		margin: 0;
 	}
```

Comment:
I think this `:last-child` block should be inside the `${from.mobileLandscape}` block. Do you agree?